Stylesheets/components/badge.css

```diff
@@ -15,59 +20,57 @@
   position: relative;
 }
 
-.ml {
+.mail {
   left: 4rem;
   bottom: 6rem;
 }
 
 .cart {
-  left: 12.5rem;
-  bottom: 8rem;
+  left: 11rem;
+  bottom: 7.8rem;
 }
 .phone {
-  left: 19.7rem;
+  left: 18rem;
   bottom: 10rem;
 }
 .insta {
-  left: 27.3rem;
+  margin-left: 4rem;
   bottom: 12rem;
 }
 
 .hero-img {
   height: 100px;
   width: auto;
   border-radius: 50%;
-  margin: 2rem;
 }
 
 .social {
-  margin: 2rem;
-  height: 30px;
-  width: 30px;
-  border: 2px solid white;
+  height: 2rem;
+  width: 2rem;
+  border: 0.2rem solid white;
   border-radius: 50%;
-  position: relative;
+  position: absolute;
```

Comment:
You have applied here correctly...but have not applied position relative to parent div

Reply:
yes this is inside parent div hero, i have changed position relative to it.